### ISSUE TITLE
Automatická dokumentace

### ISSUE THREAD

**Issue by trnkatomas** [pirati-web/urad.pirati.cz#5]
Vygerování dokumentace pomocí ApiGen (ideálně něčím podobným, ale jiným, co není tak rozbité)

**Comment by TomasVotruba** [comment]
Tohle mnohem líp řeší open-sourced Github repo

**Comment by trnkatomas** [comment]
no úplně bych to nezatracoval, třeba v nette jsem byl mnohdy rád, že se může podívat i jinam než do zdrojáku, ale možná to souvisí s velikostí projektu

**Comment by TomasVotruba** [comment]
Jo, protože Nette má strašně špatnou dokumentaci, tam to smysl dává.
